trust-packages/debian/Containerfile

```diff
@@ -12,17 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM docker.io/library/debian:11-slim as debbase
-
-ARG EXPECTED_VERSION
-ARG VERSION_SUFFIX
-
-WORKDIR /work
-
-COPY ./build.sh /work/build.sh
-
-RUN /work/build.sh $EXPECTED_VERSION $VERSION_SUFFIX /work/package.json
```

Comment:
note: this is moved, not removed - it's down below now, so we can reuse the `gobuild` container for installing validate-trust-package!